src/v/cluster/controller_log_limiter.cc

```diff
@@ -51,6 +56,34 @@ void group_limiter::update_capacity() {
     }
 }
 
+void group_limiter::setup_public_metrics() {
+    if (config::shard_local_cfg().disable_public_metrics()) {
+        return;
+    }
+
+    namespace sm = ss::metrics;
+    auto group_label = ssx::metrics::make_namespaced_label("cmd_group");
+    const std::vector<sm::label_instance> labels = {group_label(_group_name)};
+    namespace sm = ss::metrics;
+    _public_metrics.add_group(
+      "cluster:controller_limit",
```

Comment:
I think this name is not allowed by prometheus. Run it through `prometheus_sanitize::metrics_name` or replace the `:` with a `_`.